msal/src/main/java/com/microsoft/identity/client/TokenParameters.java

```diff
@@ -42,13 +43,15 @@
     private ClaimsRequest mClaimsRequest;
     private AccountRecord mAccountRecord;
     private AuthenticationScheme mAuthenticationScheme;
+    private String mCorrelationId;
```

Comment:
Is it possible that these could be final?

Reply:
Nope, these have setters and can get modified after initialization: https://github.com/AzureAD/microsoft-authentication-library-for-android/blob/dev/msal/src/main/java/com/microsoft/identity/client/TokenParameters.java#L80-L134